src/components/App/App.tsx

```diff
@@ -52,6 +51,21 @@ import { TopBar } from '../TopBar';
 import { VBrowser } from '../VBrowser';
 import { VideoChat } from '../VideoChat';
 import { getCurrentSettings } from '../routes/Settings';
+import firebase from 'firebase';
+import 'firebase/auth';
+
+// Your web app's Firebase configuration
+var firebaseConfig = {
+  apiKey: 'AIzaSyA2fkXeFokJ-Ei_jnzDso5AmjbIaMdzuEc',
```

Comment:
Should this go to the env vars?

Reply:
Probably. It's a bunch of config to add though. Maybe just make the whole thing a JSON string.